spec/linter-rubocop-spec.js

```diff
@@ -57,11 +55,11 @@ describe('The RuboCop provider for Linter', () => {
 
       waitsForPromise(() =>
         lint(editor).then((messages) => {
-          expect(messages[0].type).toEqual('Error');
-          expect(messages[0].html).toBe(msgText);
-          expect(messages[0].text).not.toBeDefined();
-          expect(messages[0].filePath).toEqual(badPath);
-          expect(messages[0].range).toEqual([[1, 6], [1, 7]]);
+          expect(messages[0].severity).toEqual('error');
```

Comment:
That won't work though since the ranges _must_ be `toEqual` (since the objects won't have the same ID as the object you are defining in the specs).